### ISSUE TITLE
Mise à jour de la balise <title> pour le formulaire d'avis, avec les nouveaux titres - nom de l'onglet

### ISSUE THREAD

**Issue by lucaa** [DISIC/wikidemarches#1046]
Il s'agit des nouveaux titres spécifiés par #1045 
Les valeurs de la balise <title> du document html doivent aussi suivre les titres h1 affichés sur la page.

Le nom de la démarche devrait aussi suivre le nom de l'étape, en plus des titres du h1.

**Comment by lucaa** [comment]
@AnthonyBrunelli  , @ugodtn : pouvez vous confirmer que cette tâche est à faire et aviser sur la nécessité d'avoir le nom de la démarche dans le titre?

**Comment by lucaa** [comment]
Le ticket #1043 est liée à cette tâche, car elle implique aussi la mise à jour de la même balise pour le formulaire, mais différemment.

**Comment by AnthonyBrunelli** [comment]
Oui, à faire, mais pas besoin de mettre/garder le nom de la démarche dans le title.

**Comment by AnthonyBrunelli** [comment]
C'est bon pour moi. @ugodtn tu peux confirmer? https://voxusagers.numerique-staging.devxwiki.com/Demarches/1897?&view-mode=formulaire-avis&nd_mode=en-ligne-enti%C3%A8rement&nd_source=button&key=9f6d2a5cc93b91073224efed38082d1e

**Comment by AnthonyBrunelli** [comment]
Ok vu avec Ugo
